Objective-C/CBLReplicator.mm

```diff
@@ -471,14 +479,26 @@ - (void) setProgressLevel: (CBLReplicatorProgressLevel)level {
 - (BOOL) isDocumentPending: (NSString *)documentID
                 collection: (CBLCollection *)collection
                      error: (NSError**)error {
+    CBLAssertNotNil(collection);
     
-    // TODO: Add implementation 
+    CBLStringBytes name(collection.name);
+    CBLStringBytes scopeName(collection.scope.name);
+    C4CollectionSpec c4spec = { .name = name, .scope = scopeName };
```

Comment:
Optionally, you could made an internal properly to get the collection spec from the collection object (e.g. collection.spec).